### PR TITLE
feat: bridge-webhook

### PR DESCRIPTION
**Pull request by thomas779**
**SUMMARY**
1. Receive a webhook event from bridge
2. Verify the authenticity of the webhook request
3. Process the payload (-> only necessary for KYC links)

**MISC**
1. fix spelling mistakes
2. add missing package so that @almithani can correctly load the env
3. fix phone number regex

**RESOURCES**
Please read [Bridge.xyz documentation](https://withbridge.notion.site/Bridge-Webhooks-User-Guide-491f603997194c868b0600a5f45051e6) re: implementation

### CODE REVIEW

**Review by almithani — APPROVED** [review]
Looks good!

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -32,4 +34,33 @@ export const utils = {
...
     return name + ' ' + surname;
   },
+  verifySignature(
+    timestamp: string,
+    rawBody: string | unknown,
+    encodedSignature: string
+  ) {
+    // Generate SHA256 digest of timestamp and raw body
+    const hash = createHash('sha256');
+    hash.update(timestamp + '.' + rawBody);
```
[the remark this answers is not in the document]
Comment (thomas779) [reply src/helpers/utils.ts:44]: That's just pulled from their docs (Bridge). I ran an example proof and it worked on my side :)

File: src/v1/handlers/bridge.ts [hunk 2]
```diff
@@ -30,7 +37,33 @@ export async function getPrefundedAccountBalance(
...
+/** @docs https://withbridge.notion.site/Bridge-Webhooks-User-Guide-491f603997194c868b0600a5f45051e6 */
+export async function processWebhooksHandler(
+  req: FastifyRequestTypebox<typeof BridgeWebhookSchema>,
+  rep: FastifyReplyTypebox<typeof BridgeWebhookSchema>
+): Promise<void> {
+  try {
+    const payload = req.body as BridgeWebhookPayload_KycLink;
+
+    switch (payload.event_type) {
+      case 'kyc_link.updated.status_transitioned':
```
[the remark this answers is not in the document]
Comment (thomas779) [reply src/v1/handlers/bridge.ts:55]: Everything is API-based on Bridge's side. So I need to manually make up the types based on their docs. The goal of this webhook is to receive KYC status updates (i.e. `kyc_link.updated.status_transitioned`) and update our internal DB based on that status change. We only care about this and nothing else.

[the remark this answers is not in the document]
Comment (thomas779) [reply src/v1/handlers/bridge.ts:55]: I like your approach of doing event-type enums. I just didn't think it was necessary to do so if we only care about 1 event type. That may change if our requirements change.

**Pull request merged**